Respawn.DatabaseTests/OracleTests.cs

```diff
@@ -1,6 +1,5 @@
 ﻿using Xunit.Abstractions;
 
-#if ORACLE
```

Comment:
This is still required because of problems with the Oracle container.

Reply:
Reintroduced Oracle compilation flag